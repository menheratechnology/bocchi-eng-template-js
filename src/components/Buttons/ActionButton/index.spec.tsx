import { render } from '@testing-library/react'

import { ActionButton } from '.'

const setup = () => {
  const defaultProps = {
    complete: false,
    onClick: jest.fn(),
  }
  const utils = render(<ActionButton {...defaultProps}>test</ActionButton>)
  return utils
}

describe('ActionButton', () => {
  it('should render', () => {
    const { asFragment } = setup()
    expect(asFragment()).toMatchSnapshot()
  })
})
