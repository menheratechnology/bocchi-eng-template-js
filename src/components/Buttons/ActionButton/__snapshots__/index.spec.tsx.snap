// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionButton should render 1`] = `
<DocumentFragment>
  <button
    class="actionButton"
    style="background-color: rgb(255, 204, 102); border: 1px solid #ffcc66;"
  >
    test
  </button>
</DocumentFragment>
`;
